R/grid_search.R

```diff
@@ -138,3 +139,27 @@ plot.sircovid_scan <- function(x, ..., what = "likelihood") {
 }
 
 
+##' @export
+plot.sample_grid_search <- function(x, ..., what = "ICU") {
+  
+  # what are we plotting
+  
+  if (what == "ICU") {
+    
+    index <- c(odin_index(x$inputs$model)$I_ICU) - 1L
+    ylab <- "ICU"
+    particles <- apply(x$trajectories[, index, ], c(1, 3), sum, na.rm = TRUE)
```

Comment:
Is this type-safe? See https://github.com/mrc-ide/sircovid-sandbox/pull/1#discussion_r402782849
i.e. will this definitely be numeric